run-make-check.sh

```diff
@@ -42,15 +62,18 @@ function run() {
         $DRY_RUN sudo /sbin/sysctl -q -w fs.aio-max-nr="${aiomax}"
     fi
 
+    CTEST_RESOURCE_FILE=`mktemp /tmp/ctest-resource-XXXXXX`
```

Comment:
could let `gen_ctest_resource_file()` `mktemp ...` and `echo` the filename. simpler this way, something like:

```sh
function gen_ctest_resource_file () {
  local filename=`mktemp /tmp/ctest-resource-XXXXXX`
  # write to $filename
  echo "$filename"
}

# ...
CHECK_MAKEOPTS+=" --resource-spec-file $(gen_ctest_resource_file)"
```